controllers/vgmanager_daemonset.go

```diff
@@ -172,6 +172,7 @@ func newVGManagerDaemonset(lvmCluster *lvmv1alpha1.LVMCluster, namespace string,
 	}
 	labels := map[string]string{
 		"app":                   VGManagerUnit,
```

Comment:
Are we using this existing label anywhere? If not I would like to remove it.

Reply:
can be removed